Provider/Pool.php

```diff
@@ -68,8 +77,8 @@ public function addProvider($name, MediaProviderInterface $instance)
     }
 
     /**
-     * @param string                                                 $name
-     * @param \Sonata\MediaBundle\Security\DownloadStrategyInterface $security
+     * @param string                    $name
+     * @param DownloadStrategyInterface $security
      */
     public function addDownloadSecurity($name, DownloadStrategyInterface $security)
```

Comment:
i think we should rename ``$security`` to ``$strategy`` in a separate PR

and we then need BC handling for ``protected $downloadSecurities``